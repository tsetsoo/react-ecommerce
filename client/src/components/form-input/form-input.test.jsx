import { shallow } from 'enzyme'
import React from 'react'
import FormInput from './form-input.component'

it('expect to render FormInput component', () => {
    expect(shallow(<FormInput />)).toMatchSnapshot();
})