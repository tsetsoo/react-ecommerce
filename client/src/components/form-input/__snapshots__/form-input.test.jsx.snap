// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render FormInput component 1`] = `ShallowWrapper {}`;
